### PR TITLE
magento/devdocs#: Disabling plugins. Add an example about a particular case.

### PR DESCRIPTION
**Pull request by atwixfirster**
## Purpose of this pull request

This pull request (PR) adds an example how to correctly disable a plugin which specified with a backslash:

```
<type name="\Magento\Catalog\Controller\Adminhtml\Product\Action\Attribute\Save">
    ....
``` 

## Affected DevDocs pages

-  https://devdocs.magento.com/guides/v2.3/extension-dev-guide/plugins.html

## Links to Magento source code

-  ...

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: src/guides/v2.3/extension-dev-guide/plugins.md [hunk 1]
```diff
@@ -485,6 +485,33 @@ Plugins can be disabled in a `di.xml` file. To disable a plugin, set the `disabl
...
+
+then you should use the next approach
+
+```xml
+<type name="\Magento\Catalog\Controller\Adminhtml\Product\Action\Attribute\Save">
+    <plugin name="bindAttributeIndexerMassUpdate" disabled="true"/>
+</type>
+```
+
+The approach with `Magento\Catalog\Controller\Adminhtml\Product\Action\Attribute\Save` will not work:
```
Comment (rogyar) [inline src/guides/v2.3/extension-dev-guide/plugins.md:507]: Please, add a minor clarification here:

```suggestion
The following example with no trailing backslash 
 `Magento\Catalog\Controller\Adminhtml\Product\Action\Attribute\Save` will not work:
```

Comment (atwixfirster) [reply src/guides/v2.3/extension-dev-guide/plugins.md]: **implemented**

Thanks, @rogyar

File: src/guides/v2.3/extension-dev-guide/plugins.md [hunk 2]
```diff
@@ -485,6 +485,33 @@ Plugins can be disabled in a `di.xml` file. To disable a plugin, set the `disabl
...
+<type name="\Magento\Catalog\Controller\Adminhtml\Product\Action\Attribute\Save">
+    <plugin name="bindAttributeIndexerMassUpdate"
+            type="\Magento\GoogleShoppingAds\Plugin\Catalog\Controller\Adminhtml\Product\Action\Attribute\Save\BindAttributeIndexer"/>
+</type>
+```
+
+then you should use the next approach
+
+```xml
+<type name="\Magento\Catalog\Controller\Adminhtml\Product\Action\Attribute\Save">
```
Comment (dobooth) [inline src/guides/v2.3/extension-dev-guide/plugins.md:502]: Is there any difference between this example and the existing example above your change? You always have to use the same class name, no?

File: src/guides/v2.3/extension-dev-guide/plugins.md [hunk 3]
```diff
@@ -485,6 +485,33 @@ Plugins can be disabled in a `di.xml` file. To disable a plugin, set the `disabl
...
+    <plugin name="bindAttributeIndexerMassUpdate" disabled="true"/>
+</type>
+```
+
+The following example with no trailing backslash `Magento\Catalog\Controller\Adminhtml\Product\Action\Attribute\Save` will not work:
```
Comment (dobooth) [inline src/guides/v2.3/extension-dev-guide/plugins.md:507]: Do you mean the leading backslash?

File: src/guides/v2.3/extension-dev-guide/plugins.md (hunk 2 above)
Comment (atwixfirster) [reply src/guides/v2.3/extension-dev-guide/plugins.md:502]: Hi, @dobooth !

`\Magento\Catalog\Controller\Adminhtml\Product\Action\Attribute\Save` is not the same as `Magento\Catalog\Controller\Adminhtml\Product\Action\Attribute\Save`.

It is the same if you compare "Donald" and "Alex".

File: src/guides/v2.3/extension-dev-guide/plugins.md (hunk 3 above)
Comment (atwixfirster) [reply src/guides/v2.3/extension-dev-guide/plugins.md:507]: > Do you mean the leading backslash?

yep :)

File: src/guides/v2.3/extension-dev-guide/plugins.md (hunk 2 above)
Comment (dobooth) [reply src/guides/v2.3/extension-dev-guide/plugins.md:502]: The example in L480 says to add the `disabled='true'` attribute to disable the plugin. It seems that the example here on L502 is doing the same thing.  If there is no difference in what we are showing, there is no need to duplicate the example. And we can probably just make a note about the importance of the leading slash. A code example of a leading slash is likely overkill.

Comment (atwixfirster) [reply src/guides/v2.3/extension-dev-guide/plugins.md:502]: @dobooth , let me please clarify.

The idea of my PR is: show developer a case when a plugin can be set on a particular class with a leading `backslash`.

In the example I've used a real code of real module.

For example we have a class `Magento\Catalog\Controller\Adminhtml\Product\Action\Attribute\Save`.

Developer can define a plugin for this class as:

```
<type name="Magento\Catalog\Controller\Adminhtml\Product\Action\Attribute\Save">
  <plugin name="plugin_a" ....>
....
```

or 

```
<type name="\Magento\Catalog\Controller\Adminhtml\Product\Action\Attribute\Save">
  <plugin name="plugin_b" ....>
....
```

So, if you want to disable `plugin_a` you have to call `Magento\Catalog\Controller\Adminhtml\Product\Action\Attribute\Save`. This is like an alias.

So, this is the important notice for developers which want to disable a plugin. They should refer to the original plugin declaration.

Thank you!

Comment (dobooth) [reply src/guides/v2.3/extension-dev-guide/plugins.md:502]: I see. You can define a plugin two ways: with the leading slash or without. If you want to disable the plugin, you must use the same way to disable as it was defined.

Comment (atwixfirster) [reply src/guides/v2.3/extension-dev-guide/plugins.md:502]: @dobooth , correct :)

Comment (dobooth) [reply src/guides/v2.3/extension-dev-guide/plugins.md]: Good. Let me know if my changes are clear and correct.

Comment (atwixfirster) [reply src/guides/v2.3/extension-dev-guide/plugins.md]: @dobooth , regarding to a PR purpose your changes are valid.

**Approved**

Thank you ;)

**Comment by dobooth** [comment]
running tests

**Pull request merged**